maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -159,7 +159,7 @@ class AndroidDriver(
                 }
                 shell("am start-activity -n $appId/${activity.name}")
             } else {
-                shell("monkey --pct-syskeys 0 -p $appId 1")
+                shell("am start-activity -n $appId/${manifest.launcherActivity.name}")
```

Comment:
We would like to keep `monkey` option as a fallback in case if somehow this code doesn't work. Could you perhaps wrap it in a try/catch?

Reply:
Done